tests/dir_helpers.py

```diff
@@ -285,3 +293,21 @@ def erepo_dir(tmp_path_factory, monkeypatch):
         path.dvc.close()
 
     return path
+
+
+@pytest.fixture
+def erepo_dir_no_remote_config(tmp_path_factory, monkeypatch):
```

Comment:
It's an overkill creating a fixture for single test. Any of the following would be better:
- use existing `erepo_dir`, drop upstream in a test
- change `erepo_dir` to not have upstream, fix any issues it causes (if that is not too hard)

Reply:
@Suor Def don't need that new fixture, but `change erepo_dir to not have upstream` won't help, see https://github.com/iterative/dvc/pull/3120#discussion_r367500983